index.js

```diff
@@ -67,6 +67,15 @@ let cachedRecords;
 let credentials;
 
 /**
+ * Maps between passed in user name and internal user name.
+ *
+ * @type {Object}
+ *
```

Comment:
Just noticed that there's some inconsistencies where we sometimes use a line break between `@type` and `@private` (visibility) - we should stick with one and apply it all around (separate PR, of course 😄 )